File: Python/그리디 알고리즘/1439.py
S = input()
zeroDivide = len(list(filter(lambda x: len(x) > 0, S.split('0'))))
oneDivide = len(list(filter(lambda x: len(x) > 0, S.split('1'))))

print(min(zeroDivide, oneDivide))

File: BOJ/Kotlin/그리디 알고리즘/1439.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import kotlin.math.min

fun main() = with(BufferedReader(InputStreamReader(System.`in`))){
    val S = readLine()
    val zeroDivide = S.split('0').filter { it.isNotEmpty() }.size
    val oneDivide = S.split('1').filter { it.isNotEmpty() }.size

    println(min(zeroDivide, oneDivide))
}